src/ResourceIndexer.js

```diff
@@ -91,7 +91,7 @@ export default class {
   buildRDFTypes() {
     this.indexObject['type'] = this.json['@graph']
       .filter((item) => item['@type'])
-      .filter((item) => item['@id'] === '')
+      .filter((item) => item['@id'] === '' || item['@id'] === this.uri)
```

Comment:
Can you toss a comment on/around this line like you did in your related editor commit?

Reply:
Done.